Shippo/APIResource.cs

```diff
@@ -433,6 +438,42 @@ public ShippoCollection<Manifest> AllManifests (Hashtable parameters)
 
         #endregion
 
+        #region Batch
+
+        public Batch CreateBatch (Hashtable parameters)
```

Comment:
This should take explicit arguments instead of `Hashtable` as requested in #9. Please see [java](https://github.com/goshippo/shippo-java-client/commit/93911beed4b637ffe071f82c1ac3274713aa7fb8) example for getting an idea. This is not done yet but should give you an idea.